File: dayThree/Java/QuestionThree.java
public class QuestionThree {
    public static void main(String... args){
        
        String word = "AbdulAzeez";
        
        int counter = 0;
        
        for(int index = 0; index < word.length(); index++){
            char letter = word.charAt(index);
            if(letter >= 'a' && letter <= 'z'){
                counter++;
            }
        }
        
        System.out.println("The Number of LowerCase is " + counter);
    }
}

File: dayThree/Javascript/QuestionThree.js
let word = "AbdulAzeez";

let counter = 0;

for(let index = 0; index < word.length; index++){
    letter = word.charAt(index);
    if(letter >= 'a' && letter <= 'z'){
        counter++;
    }
}

console.log("The Number of LowerCase is " + counter);
